File: 38-Python/Cliente.py
class Cliente:
    def __init__(self, codice, nome, cognome, indirizzo) -> None:
        self._codice = codice
        self._nome = nome
        self._cognome = cognome
        self._indirizzo = indirizzo
        self._noleggi = []

    def addNoleggio(self, noleggio):
        self._noleggi.append(noleggio)

    def __str__(self):
        return str(self._codice) + "\t" + self._cognome + "\t" + self._nome +"\t" + self._indirizzo + "\t" +str([str(noleggio) for noleggio in self._noleggi])

File: 38-Java/src/Cliente.java
import java.util.*;

public class Cliente {
    private int codCliente;
    private String nome, cognome, indirizzo;
    private List<Prenotazione> prenotazione;
    
    public Cliente(int codCliente, String nome, String cognome, String indirizzo) {
        this.codCliente = codCliente;
        this.nome = nome;
        this.cognome = cognome;
        this.indirizzo = indirizzo;
        prenotazione = new LinkedList<Prenotazione>();
    }

    public void addPrenotazione(Prenotazione p){
        prenotazione.add(p);
    }

    public String toString(){
        return codCliente + "\t" + cognome + "\t" + nome + "\t" + indirizzo + "\t" + prenotazione;
    }
}
